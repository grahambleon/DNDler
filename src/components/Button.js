import React from 'react';
import styled from 'styled-components';
import d20 from '../assets/d20.png'

const StyledButton = styled.img`
  text-align: center;
  margin: 10px;
  max-width: 190px;
`

const Button = (props) => {
  return(
    <StyledButton
      src={d20}
      alt='Choose your fate...'
      onClick={props.startCharacterRoll}
    />
  )
}

export default Button;
