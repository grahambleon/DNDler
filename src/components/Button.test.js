import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import Button from './Button.js'

it('renders', () => {
  shallow(<Button />)
})

it('renders correctly', () => {
  const tree = renderer
    .create(<Button />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
