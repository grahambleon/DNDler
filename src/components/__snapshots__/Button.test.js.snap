// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <img
    className="sc-bdVaJa kpUueE"
    src="d20.png"
  />
</div>
`;
